src/mem/ruby/slicc_interface/RubyRequest.hh

```diff
@@ -113,6 +113,38 @@ class RubyRequest : public Message
         }
     }
 
+
```

Comment:
Do we need a new constructor ? Just adding a `setWriteMask` function for the atomic requests should be cleaner.

Reply:
We can remove it once we unify the code for creating a request